### ISSUE TITLE
Visual Studio Code suddenly shows blank UI

### ISSUE THREAD

**Issue by jkpubsrc** [microsoft/vscode#116542]
- VS Code Version:
- OS Version:

Updated today on Ubuntu as there was a new version. Started VSCode again, here's the output:

Version: 1.53.2
Commit: 622cb03f7e070a9670c94bae1a45d78d7181fbd4
Date: 2021-02-11T11:48:44.518Z
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Linux x64 5.4.0-65-generic

Steps to Reproduce:

* Select "Open Folder" and open the directory I was working on the whole week. => Blank screen except status bar (which is put right to the very top of the window).
* Select other folder => everything works. Just this project suddenly causes VSCode to no longer show a UI. No error message, nothing. Just nothing there though the title gets updated which would indicate that files should have been loaded as expected.

Does this issue occur when all extensions are disabled?: Yes. Same result.

Here's a screenshot:

![vscode-bug](https://user-images.githubusercontent.com/25060709/107766968-93c7bd80-6d34-11eb-86da-bb03ba00232c.png)

**Comment by gjsjohnmurray** [comment]
Do you recall what version you were running before the update which caused the problem? 1.53 released last week, then 1.53.1 earlier this week, and 1.53.2 yesterday.

**Comment by jkpubsrc** [comment]
Not really. Very likely it was the 1.53 as I've a pretty fresh install of my OS and VSCode. Very likely it has not been the 1.53.1 - but there is uncertainty.

**Comment by jkpubsrc** [comment]
No I'm working with 5 open VSCode windows on this project simultaneously - one for every subfolder I need to apply changes to. Very annoying. This works perfectly - but I can't open the main project folder.

**Comment by bpasero** [comment]
Can you try to reproduce with our nightly insider builds? You can give our preview releases a try from: https://code.visualstudio.com/insiders/

**Comment by jkpubsrc** [comment]
Does the nightly builds include a command line debug switch or something so that you have some log output what the application is doing?

Unfortunately I had to perform some work today - something completely different - and did not find the time for such an experiment. Unfortunately I have to work on weekends at the moment, but I'll try to test it with the nightly build right now.

**Comment by bpasero** [comment]
@jkpubsrc thanks for trying, both versions allow you to start from the command line with an extra `--verbose` to get log output printed to the console.

**Comment by jkpubsrc** [comment]
After uninstalling/reinstalling VSCode multiple times, installing the nightly build, and now reinstalling the stable version again for an attempt to get verbose output, I do no longer get a blank screen. Now I get the screen as I should have seen in the first place. Quite strange as I have even not booted the machine since that problem occurred the first time.

Could it have been an issue with an extension that reinstalling has resolved somehow? After reinstalling I saw a quite strange message box in VSCode telling me that this freshly installed VSCode software would now need to be restarted as some files had changed on disk. That message surprised me as I did not see any reason for why that should be the case as I did not even touch the system in any way in those few seconds between VSCode being launched and displaying that message.

But now VSCode works again as expected without a blank screen.

That means: I can no longer reproduce that issue. Therefore I won't be able to provide you with more information as long as I don't encounter that problem again.

So now I see no reason why it would make sense to keep this issue open.

Thank you for your support!
